cve_bin_tool/output_engine/html.py

```diff
@@ -92,41 +98,30 @@ def output_html(
                     fix_id=hid,
                 )
 
-                # generate analysis_data
-                if cve.severity.lower() == "critical":
-                    analysis_data["critical"] += 1
-                elif cve.severity.lower() == "high":
-                    analysis_data["high"] += 1
-                elif cve.severity.lower() == "medium":
-                    analysis_data["medium"] += 1
-                elif cve.severity.lower() == "low":
-                    analysis_data["low"] += 1
-                else:
-                    pass
+            analysis_data = Counter([cve.severity for cve in product.cves])
```

Comment:
You don't need to use list comprehension here. It will materialize list before giving it to Counter constructor which is slower so use generator directly.
```suggest
            analysis_data = Counter(cve.severity for cve in product.cves)
```